packages/gatsby/src/query/file-parser.ts

```diff
@@ -393,12 +441,18 @@ async function findGraphQLTags(
 
 const cache = {}
 
-export default class FileParser {
-  constructor({ parentSpan } = {}) {
+export class FileParser {
```

Comment:
This ought to be a breaking change since the default export got replaced with a named export but no other files in this PR were changed. So either it is changing an external API, which I don't think is the case here, or something in core that depends on this is now broken...